hibp.js

```diff
@@ -70,13 +69,13 @@ const HIBP = {
       // ]
       for (const breachedAccount of response.body) {
         if (sha1.toUpperCase() === sha1Prefix + breachedAccount.hashSuffix) {
-          foundBreaches = await DBUtils.getBreachesByNames(breachedAccount.websites);
+          foundBreaches = allBreaches.filter(breach => breachedAccount.websites.includes(breach.Name));
```

Comment:
I just realized this loop doesn't break for some reason. While we're here, can you refactor this to something like:
```
if (the hash doesn't match) {
  continue;
}
foundBreaches = [...]
```

Reply:
Interesting. What's the advantage to inverting the conditional logic? Optimization?

Reply:
I thought about this some more, and I'm keeping the conditional logic like it is, so that I can `break` when I find the match.

That should achieve the same optimization of the loop.

If there's another strong reason to invert the conditional logic, let me know here and/or file an issue for follow|fix-up.